ansible_base/feature_flags/views.py

```diff
@@ -1,25 +1,41 @@
 from django.conf import settings
 from django.utils.translation import gettext_lazy as _
 from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
 
-from ansible_base.feature_flags.serializers import FeatureFlagSerializer
+from ansible_base.feature_flags.models import AAPFlag
+from ansible_base.feature_flags.serializers import FeatureFlagStatesSerializer, OldFeatureFlagSerializer
 from ansible_base.lib.utils.views.ansible_base import AnsibleBaseView
+from ansible_base.lib.utils.views.django_app_api import AnsibleBaseDjangoAppApiView
+from ansible_base.lib.utils.views.permissions import IsSuperuserOrAuditor, try_add_oauth2_scope_permission
 
 from .utils import get_django_flags
 
 
-class FeatureFlagsStateListView(AnsibleBaseView):
+class FeatureFlagsStatesView(AnsibleBaseDjangoAppApiView, ModelViewSet):
+    """
+    A view class for displaying feature flags states
```

Comment:
Might be help to explain where they should go to change them?

Makes sense that this is read-only. But it would be really really helpful to direct users to where they need to be.

Reply:
Great note, I will put a note here to guide folks to the readme, which contains the instructions about this.